src/plugins/tables/tables.js

```diff
@@ -84,6 +84,29 @@ var componentName = "wb-tables",
 						dataTableExt = $.fn.dataTableExt;
 
 					/*
+					* Add printing support
+					*/
+					if ( $elm.hasAttribute( "print" ) ) {
+						var $btn = $( "<button class='btn btn-default mrgn-bttm-md' id='" + elmId + "_datatable-print'>Print Table</button>" );
+						$elm.before( $btn );
+						$btn.click( function() {
+							var css = "<link href='" + wb[ "/" ].replace( "js", "css" ) + "/theme" + wb.getMode() + ".css' rel='stylesheet' />", table = $( "<div>" ).append( $elm.clone() ).html(), html = css + table, printWin = window.open( "about:blank", "_blank" );
+							printWin.document.write( html );
+							printWin.document.close();
+							printWin.focus();
+							setTimeout( function() {
+								printWin.print();
+							}, 500 );
+							printWin.onfocus = function() {
+								setTimeout( function() {
```

Comment:
Do you think it's possible to hook on a DOM/Window/JQuery event for that? 
It's just because I do not know if 500 ms will be always enough, like for slow mobile or computer, or if the 500 ms is too much time for nothing.